src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+import duke.command.Command;
+import duke.exceptions.UnknownException;
+import duke.util.TaskList;
+import duke.util.Parser;
+import duke.util.Ui;
+import duke.util.Storage;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import javafx.scene.image.Image;
```

Comment:
I like the consistency of the import statements thus far. Perhaps you could remove the new lines between 'import javafx' statements to make the import statements perfect?